packages/react-composites/tests/browser/call/CallComposite.test.ts

```diff
@@ -51,7 +53,9 @@ test.describe('Call Composite E2E Configuration Screen Tests', () => {
   test('composite pages load completely', async ({ pages }) => {
     const page = pages[0];
     await stubLocalCameraName(page);
+    await disableTooltips(page);
```

Comment:
Just a note: for other tooltip issues we've used `clickOutsideOfPage` in common/utils.ts
Both solutions should work and I'm happy to keep this the way it is, just highlighting for awareness that that fn exists

Reply:
(thinking some more: an alternative solution would be to add `clickOutsideOfPage` to waitForCallCompositeToLoad so new tests for the configuration page shouldn't hit this be default)

Reply:
> Just a note: for other tooltip issues we've used `clickOutsideOfPage` in common/utils.ts Both solutions should work and I'm happy to keep this the way it is, just highlighting for awareness that that fn exists

Basis on your comment in the bug https://skype.visualstudio.com/SPOOL/_workitems/edit/2826640
Looks like clickOutsideOfPage doesn't always work?

Reply:
Lol! Looks like clickOutsideOfPage is not being used in these tests
But yeah I think what I was getting at is it's tough when someone adds something new we suddenly might have flakiness from tooltips which the OG developer shouldn't have to think about... So maybe we use your functions and disable tooltips by default and only enable them when testing? But we can cross that bridge another time

Reply:
It's confusing to have two ways to do things.

Let's pick one of the two ways to disable tooltips and follow that uniformly.